database/migrations/2017_03_13_212917_CreateSharedExpenseTable.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSharedExpenseTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sharedexpenses', function (Blueprint $table) {
+            $table->unsignedInteger('expense_id');
```

Comment:
this expense_id is a foreign key to the id of expense table. it shouldn't be increments